### PR TITLE
Update CONTRIBUTING devdocs

### PR DESCRIPTION
**Pull request by mdtusz**
Gotta stay fresh 🍃 

I'll add a gif too but I don't want to pollute the repo with it so I'll add it directly so github can host it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,15 +63,30 @@ tab in your browser.
 
 **Step 3** Open up the console and start developing
 
-Make some modifications to the source, refresh the page and check the results
-by for example pasting in the console:
+A typical workflow is to make some modifications to the source, update the
+test dashboard, inspect and debug the changes, then repeat. The test dashboard
+comes bundled with some useful tools while developing - all bundled under the
+`Tabs` object:
```
Comment (etpinard) [inline CONTRIBUTING.md]: maybe add a link to [`devtools.js`](https://github.com/plotly/plotly.js/blob/master/devtools/test_dashboard/devtools.js) ?

**Comment by etpinard** [comment]
> I'll add a gif too but I don't want to pollute the repo with it so I'll add it directly so github can host it.

:+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (mdtusz) [inline CONTRIBUTING.md]: Good call.

**Comment by etpinard** [comment]
:dancer:

**Pull request merged**